ploneorg/core/profiles/default/workflows/foundation_member_workflow/definition.xml

```diff
@@ -136,23 +162,34 @@
              title="Member renews membership"
              new_state="approved" trigger="USER"
              before_script="" after_script="">
-  <action url="%(content_url)s/view" category="workflow">Renew</action>
+  <action url="%(content_url)s/content_publish_form"
+          category="workflow">Renew</action>
   <guard>
-   <guard-permission>Modify portal content</guard-permission>
+   <guard-permission>Request review</guard-permission>
+  </guard>
+ </transition>
+ <transition transition_id="suspend"
+             title="Suspend for Renewal"
+             new_state="pending_renewal" trigger="USER"
+             before_script="" after_script="">
+  <action url="%(content_url)s/content_publish_form"
+          category="workflow">Suspend for Renewal</action>
+  <guard>
+   <guard-permission>Foundation Member: Manage Members</guard-permission>
```

Comment:
Above in line 18 we are using the Manager role. Wondering if you want to be consistently using one or the other.

Reply:
I think I has originally put in the Manager role for this, and it didn't work for some reason. This one worked, so I left it.